.gitignore

```diff
@@ -1,9 +1,8 @@
 .idea/*
+.vscode
```

Comment:
Not sure how I feel about this. Some of these files are clearly intended to be included in VCS, but I haven't found them useful in this repo.

Reply:
You can have whatever you want on your local machine but when it comes to sharing file-level configuration we have cross-editor solution: https://github.com/confio/ts-relayer/blob/main/.editorconfig

Reply:
Yeah I'm a big fan of editorconfig, but it doesn't really cover the same things here, eg:
- recommended extensions only makes sense if you share the file with others, eg via committing it to git (but I personally found it annoying that VSCode would recommend them every time)
- in my python/poetry repos I've found it useful to set the python path to the local installation in `settings.json` (but not so relevant for Node/npm repos)